lib/capybara/rspec/features.rb

```diff
@@ -21,9 +23,12 @@ def self.included(base)
           alias :background :before
           alias :scenario :it
           alias :xscenario :xit
+          alias :fscenario :fit
```

Comment:
does RSpec 2.X has `fit` or `fdescribe` aliases? 

http://www.relishapp.com/rspec/rspec-core/v/2-14/docs/configuration/alias-example-to